### DISCUSSION TITLE
Showing accordion collapsed by default

### DISCUSSION THREAD

**Discussion by EeKay** [orchidsoftware/platform#1866] [Q&A]
Hey @tabuna,

I was wondering the framework provides a way to make an accordion on a form collapsed by default (so it's only expanded when a user clicks on it).

Want to use this for mutation details (timestamps, last edited by, ..). Fields that are only interesting when you need them.

**Comment by EeKay** [comment]
I've noticed on the example website you added a "Click for me!" line on the [layouts example page](https://demo.orchid.software/example-layouts). This shows/hides the layout items first name, last name, username....

I can't find this in the github code, but this is exactly what I was looking for ... how can I achieve this?

**Reply by tabuna** [reply]
The demo uses version 9.x in 10.x this layer was removed because I don't use it and don't want to support it. You can copy it to your project https://github.com/orchidsoftware/platform/blob/9.x/src/Screen/Layouts/Collapse.php

**Reply by EeKay** [reply]
Would it work if accordeon got an option "startCollapsed" so it would render closed at first glance?
